index.js

```diff
@@ -238,7 +240,8 @@ client.on("message", message => {
 	}
 });
 
-client.on("messageUpdate", (oldMessage, newMessage) => {
```

Comment:
did you make sure this works? looks like something where it gives you back 2 values and its the old and new, if thats the case this would make it think the new is just the same no?